addons/website/migrations/13.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,139 @@
+---Models in module 'website'---
+obsolete model website.redirect (renamed to website.rewrite)
+new model website.rewrite (renamed from website.redirect)
+# DONE: pre-migration: renamed models
+
+new model website.route
+new model website.track
+new model website.visitor
+# NOTHING TO DO
+
+---Fields in module 'website'---
+website      / ir.ui.view               / track (boolean)               : NEW hasdefault
+website      / mrp.document             / key (char)                    : previously in module mrp
+website      / mrp.document             / website_id (many2one)         : previously in module mrp
+website      / mrp.document             / website_url (char)            : previously in module mrp
+website      / res.partner              / is_published (boolean)        : previously in module website_partner
+website      / res.partner              / visitor_ids (many2many)       : NEW relation: website.visitor
+website      / res.partner              / website_published (boolean)   : previously in module website_partner
+website      / res.partner              / website_url (char)            : previously in module website_partner
+website      / res.users                / is_published (boolean)        : previously in module website_partner
+website      / res.users                / website_published (boolean)   : previously in module website_partner
+website      / res.users                / website_url (char)            : previously in module website_partner
+website      / website                  / default_lang_code (char)      : DEL
+website      / website                  / logo (binary)                 : NEW attachment: True, hasdefault
+website      / website                  / social_googleplus (char)      : DEL
+website      / website.menu             / group_ids (many2many)         : NEW relation: res.groups
+website      / website.menu             / mega_menu_classes (char)      : NEW
+website      / website.menu             / mega_menu_content (html)      : NEW
+website      / website.rewrite          / route_id (many2one)           : NEW relation: website.route
+# NOTHING TO DO
+
+website      / website.rewrite          / name (char)                   : NEW required
+# DONE: post-migration: filled name with some heuristic name

```

Comment:
This should be done in pre-migration pre-creating the column, for avoiding the constraint error.